tools/const_finder/lib/const_finder.dart

```diff
@@ -64,12 +60,14 @@ class _ConstVisitor extends RecursiveVisitor<void> {
 
   @override
   void visitConstructorInvocation(ConstructorInvocation node) {
-    final Class parentClass = node.target.parent as Class;
+    final Class parentClass = node.target.parent! as Class;
     if (_matches(parentClass)) {
+      assert(node.location != null);
```

Comment:
Nit: this assertion can be removed (checked at the next line anyway).

Reply:
Removed